distribution/lib/Standard/Base/0.0.0-dev/src/Data/Text/Prim_Text_Helper.enso

```diff
@@ -4,4 +4,4 @@
 
    Forces flattening of a text value.
 optimize : Text
-optimize = @Builtin_Method "Prim_Text_Helper.optimize"
+optimize text = @Builtin_Method "Prim_Text_Helper.optimize"
```

Comment:
presumably should be `self`.

Reply:
Looking at the (only) usage, I think it is good as-is:

![image](https://user-images.githubusercontent.com/1436948/180425012-c393cad4-3752-4b03-8a91-5e461cb64990.png)

This is not an extension function on Text.